lib/default-options.js

```diff
@@ -16,7 +16,8 @@ module.exports = function defaultOptions () {
             'assert.notDeepEqual(actual, expected, [message])',
             'assert.deepStrictEqual(actual, expected, [message])',
             'assert.notDeepStrictEqual(actual, expected, [message])'
-        ]
+        ],
+        additionalMethods: []
```

Comment:
- `patterns`: methods we are intended to be enhanced
- `additionalMethods`: methods we are __not__ intended to be enhanced

A little symmetry breaking here?
I want a intention revealing name for the latter pattern.